packages/core/src/components/Chart/Chart.js

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import * as Classes from '../../common/classes'
+
+const CHART_URL = 'https://bitfinexcom.github.io/bfx-hf-tradingview'
+
+const DEFAULT_THEME = 'default-theme:light-mode'
+
+const Chart = ({
+  market: {
+    wsID,
+    base,
+    quote,
+  },
+  theme,
+}) => {
+  const queryString = new URLSearchParams({
+    wsID,
+    base,
+    quote,
```

Comment:
ufx-ui/core is meant to be work with any data and should not depend on bitfinex only endpoints or selectors.
If chart-iframe can only work with bitfinex endpoints, lets not add component in ufx-ui/core and only keep this code in ufx-ui/bfx-containers/Chart. i.e. lets only add `container` comp for Chart.
Else if chart-iframe can work with any third party endpoints. keep queryString as prop here, and build queryString in Chart.container by using env from config.selectors